setup.cfg

```diff
@@ -29,6 +29,7 @@ packages = find:
 install_requires =
     numpy >= 1.6.1
     pandas >= 0.14
+    pint >= 0.18
```

Comment:
We will also need an entry in requirements/pymeasure.yml

Reply:
which version? I see discrepancies between setup.cfg and pymeasure.yml.

Reply:
pymeasure.yml gives _specific_ fixed versions that e.g. our CI is using.
setup.cfg gives _minimum_ versions (because you make use of a feature/rely on a fix that only appeared at some point. 
Speaking of which, is there a reason for your minimum of 0.18, or could it be just `pint`? If the latter, that is preferable as it aids package compatibility resolution when installing many into an environment.

Reply:
I just used the latest version which is still compatible with python 3.7

Reply:
> I just used the latest version which is still compatible with python 3.7, which is 0.18.

I see. In this case just use plain `pint`. 
As we still want to support python 3.7, we can't demand pint >=0.19 as proposed above, so use 0.18 in pymeasure.yml

>  What are the criteria to choose a version?
for the CI testing yml file, I simply use the current version at a time when I add a dependency. We can then update the versions manually from time, or when incompatibilities arise (e.g. because the numpy version becomes very old, or somesuch).